spec/invoice_repository_spec.rb

```diff
@@ -0,0 +1,123 @@
+require 'rspec'
+require './lib/invoice'
+require './lib/invoice_repository'
+
+describe InvoiceRepository do
+  describe '#initialize' do
+    it 'is an instance of InvoiceRepository' do
+      path = './data/invoices.csv'
+      inre = InvoiceRepository.new(path)
+
+      expect(inre).to be_an_instance_of InvoiceRepository
+    end
+  end
+
+  describe '#all' do
+    it 'returns an array of all invoice instances' do
+      path = './data/invoices.csv'
+      inre = InvoiceRepository.new(path)
+
+      expect(inre.all).to be_an Array
+      expect(inre.all.first).to be_an_instance_of Invoice
+      expect(inre.all.first.status).to eq("pending")
```

Comment:
We probably don't need to test accessing attributes of `Invoice` if we've already shown that we can access the `Invoice`, but better to test too much than not enough